### ISSUE TITLE
Error when rendering

### ISSUE THREAD

**Issue by ashtyn3** [jacoborus/estilo#48]
I get the following error when I try to render my theme.
```
Error:

TypeError: hexterm is not a function
    at /home/ashtyn372/.config/nvm/versions/node/v14.10.1/lib/node_modules/estilo/src/compile-palette.js:20:41
    at Array.forEach (<anonymous>)
    at module.exports (/home/ashtyn372/.config/nvm/versions/node/v14.10.1/lib/node_modules/estilo/src/compile-palette.js:8:23)
    at /home/ashtyn372/.config/nvm/versions/node/v14.10.1/lib/node_modules/estilo/src/render-project.js:32:19
    at Array.forEach (<anonymous>)
    at module.exports (/home/ashtyn372/.config/nvm/versions/node/v14.10.1/lib/node_modules/estilo/src/render-project.js:31:28)
    at /home/ashtyn372/.config/nvm/versions/node/v14.10.1/lib/node_modules/estilo/src/load-project.js:39:22
```

**Comment by Tintef** [comment]
@ashtyn3 I fixed it by manually adding [the solution on this issue](https://github.com/jacoborus/estilo/issues/45)

**Comment by jacoborus** [comment]
Fixes in #46
